addons/hw_drivers/websocket_client.py

```diff
@@ -44,18 +43,18 @@ def on_message(ws, messages):
         Synchronously handle messages received by the websocket.
     """
     messages = json.loads(messages)
-    _logger.debug("websocket received a message: %s", pprint.pformat(messages))
     iot_mac = helpers.get_mac_address()
     for message in messages:
         message_type = message['message']['type']
+        _logger.info("Received message of type %s", message_type)
         if message_type == 'iot_action':
             payload = message['message']['payload']
             if iot_mac in payload['iotDevice']['iotIdentifiers']:
                 for device in payload['iotDevice']['identifiers']:
                     device_identifier = device['identifier']
                     if device_identifier in main.iot_devices:
                         start_operation_time = time.perf_counter()
-                        _logger.debug("device '%s' action started with: %s", device_identifier, pprint.pformat(payload))
+                        _logger.info("device '%s' action started", device_identifier)
```

Comment:
```suggestion
                        _logger.info("device '%s' action started", device_identifier)
                        _logger.debug("action payload: %s", pprint.pformat(payload))
```
payload content can be useful on some support cases, I think it is better to show it only on debug mode